epub33/a11y/index.html

```diff
@@ -764,62 +764,63 @@ <h6>Page Breaks</h6>
 						<h5>Overview</h5>
 
-						<p>Media Overlays provide an accessible playback experience for anyone who benefits from having
-							text and audio synchronized. They are also useful to users who only require audio playback,
-							or only benefit from reading with text highlighting. Media Overlays also enable a seamless
-							playback experience from beginning to end of an EPUB Publication for all these users.</p>
-
-						<p>The most basic <a href="https://www.w3.org/TR/epub/#sec-overlays-def">Media Overlay
-								Documents</a> [[EPUB-3]] provide only minimal instructions to Reading Systems, however.
-							They indicate the text to highlight and the audio clip that corresponds to the text. The
-							result is that users only have basic start and stop options available.</p>
-
-						<p>EPUB Creators need to add structure and semantics to Media Overlay Documents to allow Reading
-							Systems to present more usable experiences. With richer markup, a Reading System could
-							provide the ability to skip past secondary content that interferes with the primary
-							narrative, escape users from deeply nested structures like tables, and allow them to
-							navigate through the sections of the publication without having to go to the table of
-							contents.</p>
-
-						<p>Adding structure and semantics to Media Overlay Documents broadly falls under the objective
-							of the <a href="https://www.w3.org/TR/WCAG2/#info-and-relationships">Info and Relationships
-								success criterion</a> [[WCAG2]]. Without structured and semantically meaningful playback
-							sequences, the effect is to deprive users of rich navigation of the content.</p>
+						<p>The provision of synchronized text-audio playback helps address various user needs. It not
+							only enables a seamless visual and auditory reading experience from beginning to end of an
+							EPUB Publication, but is useful to users who only require audio playback (e.g., who cannot
+							see the text or are prevented from reading visually due to motion-sickness) or who only
+							benefit from reading with text highlighting (e.g., readers with dyslexia).</p>
+
+						<p>The most basic forms of synchronized text-audio playback provide only minimal instructions to
+							Reading Systems, however. They indicate the text to highlight and the audio clip that
+							corresponds to the text. The result is that users only have basic start and stop options
+							available.</p>
+
+						<p>EPUB Creators need to add additional structure and semantics to allow Reading Systems to
+							present more usable experiences. With greater context, a Reading System can provide the
+							ability to skip past secondary content that interferes with the primary narrative, escape
+							users from deeply nested structures like tables, and allow them to navigate through the
+							sections of the publication without having to go to the table of contents.</p>
```

Comment:
I'm not sure how it would work. Sounds mixed up with ARIA landmark roles.

We don't even have a way of identifying headings in overlays, so it'd sort of be like escapability at a nebulous "section" level. I'm going to strip that bit in the next commit.